.github/workflows/tag-merged-pr.yml

```diff
@@ -0,0 +1,14 @@
+on:
+  pull_request:
+    branches: 
+      - master
+    types: [closed]
+
+jobs:
+  my-action:
+    if: ${{ github.event.pull_request.merged }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions-ecosystem/action-add-labels@v1
+        with:
+          labels: S-waiting-on-deploy
```

Comment:
Can we also remove all other status labels, like waiting-on-review and waiting-on-author?

Reply:
Sure!